### PR TITLE
Changed 'writefile' to 'save-file'

### PR DESCRIPTION
**Pull request by Kinnardian**
Changed `writefile` to `save-file` to be consistent with `save-table` / `write-table`.

### CODE REVIEW

File: arc.arc [hunk 1]
```diff
@@ -2898,7 +2899,7 @@ Useful in higher-order functions, or to index into lists, strings, tables, etc."
             (= (savers* ',var) (fn (,gv) (,save ,gv ,file)))))))
 
 (mac diskvar (var file)
-  `(fromdisk ,var ,file nil readfile1 writefile))
```
Comment (akkartik) [inline arc.arc]: Hmm, now I'm wondering if we should also rename `readfile1` to `load-file` and so on.

But that piles on the incompatibilities. Others may object. What do you think, @hjek?

**Comment by Kinnardian** [comment]
Breaks `temstore` https://github.com/arclanguage/anarki/blob/9e53a1f2ed01588696ec0bbddb442dcc7a0374ab/lib/tem.arc#L64

**Comment by akkartik** [comment]
Good catch! I also see a call to `writefile` in apps/news/news.arc.

Worse, none of our tests catch this. That's something to think about. I'd like to test reading/writing templates.

Could you fix both these callsites and let me know? Let's get this merged. We can always tweak or roll back things later. No need to add any tests in this PR.

**Comment by Kinnardian** [comment]
All in one new PR? I think I can pull that off.

**Comment by akkartik** [comment]
You can just add to this PR. Just do a new commit and `git push`. You should see this page update.

**Comment by akkartik** [comment]
Oh, before you do, run this:

```
$ git pull --rebase
```

**Comment by Kinnardian** [comment]
I did something wrong . . .

**Comment by akkartik** [comment]
Very strange. It looks ok on the commandline, but GitHub's UI seems to be confused.

**Comment by akkartik** [comment]
Let's try merging it and see what happens.

**Comment by Kinnardian** [comment]
It might be because my remote is not my fork:
```
$ git remote -v
origin	https://github.com/arclanguage/anarki/ (fetch)
origin	https://github.com/arclanguage/anarki/ (push)
```
¿

**Pull request merged**